src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How The Merge impacts ether supply
+description: Breakdown on how The Merge will impact ether supply
+lang: en
+sidebar: true
+---
+
+# How The Merge impacts ether supply {#how-the-merge-impacts-ether-supply}
+
+<Card
```

Comment:
- I think we need an introductory paragraph before this card.
- This card could use have a title to orient ("The pre-merge breakdown")

Reply:
Per discussion, will shift "tldr" box down below some of the intro copy, and add "ETH issuance tldr" as a title to the card. :fire: emoji removed from bullet.